extensions/wikia/ImageReview/ImageReviewEventsHooks.class.php

```diff
@@ -27,6 +43,8 @@ public static function onArticleUndelete( Title $title, $created, $comment ) {
 	public static function onArticleDeleteComplete( Page $page, User $user, $reason, $articleId ) {
 		global $wgCityId;
 
+		// TODO: should ImageReview service be notified about article delete?
```

Comment:
IMO we should send event via rabbit to image-review service and then service should remove the image from set of images to review, however I'm not sure if service is ready for this